build-parent/pom.xml

```diff
@@ -95,7 +95,7 @@
         <junit4.version>4.13.2</junit4.version>
 
         <!-- The image to use for tests that run Keycloak -->
-        <keycloak.server.version>25.0.6</keycloak.server.version>
+        <keycloak.server.version>26.0.7</keycloak.server.version>
```

Comment:
@jedla97 Hi, I believe the client version should be aligned

Reply:
The client libs have now dedicated release cycle https://www.keycloak.org/docs/26.0.0/release_notes/#dedicated-release-cycle-for-the-client-libraries so we would need to use `26.0.3` for keycloak itself as the client don't have 26.0.7 version (latest is 26.0.3). Which we using https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml#L180